src/LevelManager.cpp

```diff
@@ -0,0 +1,33 @@
+#include "LevelManager.h"
+
+void LevelManager::LoadLevel(String file) {
+	int rowCount = 0;
+	int colCount = 0;
+
+	levelFile.open(file);
+	if (levelFile.is_open()) {
```

Comment:
Same thing as the previous comments. Arrange the brackets and separate words in variable names.